views/pages/fullEpisodes.ejs

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
This page is not well-formed html - it's missing html, head and body tags. (it's therefore missing title, and any link to css, too)

index.ejs has that structure, but this EJS does not.